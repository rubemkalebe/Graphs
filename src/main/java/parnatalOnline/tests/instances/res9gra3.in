9 3
26 22 14 20 13 25 2 30
2 5 29 16 15 14 1
22 27 28 13 8 19
11 21 10 21 3
6 12 25 1
29 14 6
24 17
2
