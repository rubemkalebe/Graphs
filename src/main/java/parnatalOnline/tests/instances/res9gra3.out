29.000000
0 7, 1 2, 1 3, 1 8, 3 6, 4 5, 4 8, 7 8, 
3356640
3.810875;0.000000;